internal/testrunner/compiler_runner.go

```diff
@@ -341,7 +341,7 @@ func (c *compilerTest) verifyDiagnostics(t *testing.T, suiteName string, isSubmo
 		tsbaseline.DoErrorBaseline(t, c.configuredName, files, c.result.Diagnostics, c.result.Options.Pretty.IsTrue(), baseline.Options{
 			Subfolder:           suiteName,
 			IsSubmodule:         isSubmodule,
-			IsSubmoduleAccepted: len(c.result.Program.UnsupportedExtensions()) != 0, // TODO(jakebailey): read submoduleAccepted.txt
+			IsSubmoduleAccepted: c.containsUnsupportedOptions(), // TODO(jakebailey): read submoduleAccepted.txt
```

Comment:
You can delete my TODO; that was leftover from in the middle of my PR and then missed in review.